contracts/AlphaStakingV3.sol

```diff
@@ -0,0 +1,170 @@
+pragma solidity 0.6.12;
+
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/token/ERC20/IERC20.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/token/ERC20/SafeERC20.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/utils/ReentrancyGuard.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/math/SafeMath.sol';
+import 'OpenZeppelin/openzeppelin-contracts@3.2.0/contracts/proxy/Initializable.sol';
+
+contract AlphaStakingV3 is Initializable, ReentrancyGuard {
+  using SafeERC20 for IERC20;
+  using SafeMath for uint;
+
+  event SetWorker(address worker);
+  event Stake(address owner, uint share, uint amount);
+  event StakeFor(address staker, address owner);
+  event Unbond(address owner, uint unbondTime, uint unbondShare);
+  event Withdraw(address owner, uint withdrawShare, uint withdrawAmount);
+  event CancelUnbond(address owner, uint unbondTime, uint unbondShare);
+  event Reward(address worker, uint rewardAmount);
+  event Extract(address governor, uint extractAmount);
+  event SetMerkle(address merkle);
+
+  uint public constant STATUS_READY = 0;
+  uint public constant STATUS_UNBONDING = 1;
+  uint public constant UNBONDING_DURATION = 30 days;
+  uint public constant WITHDRAW_DURATION = 3 days;
+
+  struct Data {
+    uint status;
+    uint share;
+    uint unbondTime;
+    uint unbondShare;
+  }
+
+  IERC20 public alpha;
+  address public governor;
+  address public pendingGovernor;
+  address public worker;
+  uint public totalAlpha;
+  uint public totalShare;
+  mapping(address => Data) public users;
+  address public merkle;
+
+  modifier onlyGov() {
+    require(msg.sender == governor, 'onlyGov/not-governor');
+    _;
+  }
+
+  modifier onlyWorker() {
+    require(msg.sender == worker || msg.sender == governor, 'onlyWorker/not-worker');
+    _;
+  }
+
+  modifier onlyMerkle() {
+    require(msg.sender == merkle, 'onlyMerkle/not-merkle');
+    _;
+  }
+
+  function initialize(IERC20 _alpha, address _governor) external initializer {
+    alpha = _alpha;
+    governor = _governor;
+  }
+
+  function setWorker(address _worker) external onlyGov {
+    worker = _worker;
+    emit SetWorker(_worker);
+  }
+
+  function setPendingGovernor(address _pendingGovernor) external onlyGov {
+    pendingGovernor = _pendingGovernor;
+  }
+
+  function setMerkle(address _merkle) external onlyGov {
+    merkle = _merkle;
+    emit SetMerkle(_merkle);
+  }
+
+  function acceptGovernor() external {
+    require(msg.sender == pendingGovernor, 'acceptGovernor/not-pending');
+    pendingGovernor = address(0);
+    governor = msg.sender;
+  }
+
+  function getStakeValue(address user) external view returns (uint) {
```

Comment:
change input parameter to have underscore at first i.e. `function getStakeValue(address _user)`